Gemfile

```diff
@@ -7,7 +7,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'rails', '~> 6.1.0'
+gem 'rails', '~> 7.0.0', '>= 7.0.5.1'
```

Comment:
Make sure it matches all (most) the other apps!

i14y is using `gem 'rails', '~> 7.0.0', '>= 7.0.1'`